IdentitySdkCore/IdentitySdkCore/Classes/ReachFivePasswordLess.swift

```diff
@@ -41,10 +41,14 @@ public extension ReachFive {
     
     func verifyPasswordlessCode(verifyAuthCodeRequest: VerifyAuthCodeRequest) -> Future<AuthToken, ReachFiveError> {
         let pkce: Pkce? = storage.take(key: pkceKey)
+        guard let pkce else {
+            return Future(error: .TechnicalError(reason: "Pkce not found"))
```

Comment:
Does this change the error that is returned when PKCE is missing ? Before we unwrapped the value which throws an error. I'm not sure this is important.

Reply:
Ça ne lance pas un autre type d'erreur, ça fait carrément planter l'appli.

Reply:
Je me demandais quelle erreur était remontée (la doc dit qu'un _unwrap_ sur une option vide "lève une exception" sans plus de précision) et comment (l'exception n'est sûrement pas _wrappée_ dans une `Future`)